trie/verkle.go

```diff
@@ -115,9 +115,14 @@ func (t *VerkleTrie) GetAccount(addr common.Address) (*types.StateAccount, error
 		return nil, fmt.Errorf("GetAccount (%x) error: %v", addr, err)
 	}
 
-	if values == nil {
+	emptyAccount := true
+	for i := 0; values != nil && i <= utils.CodeHashLeafKey && emptyAccount; i++ {
+		emptyAccount = emptyAccount && values[i] == nil
+	}
+	if emptyAccount {
 		return nil, nil
 	}
+
```

Comment:
If `values == nil` the loop ends immediately and returns `nil, nil`. If `values != nil` we return `nil, nil` if _all_ first five values are empty (i.e: since other values like storage slot or contract entries might not be empty).

Note: if we change to the new EIP-4762, then `utils.CodeHashLeafKey` will change and this code will continue to work as expected, so no "TODO" here.

Reply:
great find, btw